### ISSUE TITLE
Plugin is incompatible with v2.36 on Ubuntu 16.04

### ISSUE THREAD

**Issue by diffusae** [pfn/keepasshttp#337]
With the lastest KeePass2 version (v2.36) in the repos, the plugin is incompatible on Ubuntu 16.04.
I've tried the dlls, also i've installed mono-complete, but there was no was to get it working.

Maybe you could solve the issue?

Regards

**Comment by diffusae** [comment]
Version of KeePass is 2.36
Version of KeePassHttp was the latest fom git (2.34?)
Error message: "Plugin is incompatible with the current Keypass version"
Used clients and their versions is Ubuntu 16.04

**Comment by va-an** [comment]
I have this problem too on debian testing with KeePass versions:
* 2.28
* 2.35
* 2.37

What i'm doing wrong?

**Comment by diffusae** [comment]
Yes, the problem also exists on different distributions. You didn't do something wrong. It's a problem with Http-Connector plugin. As a workaround you can use version 2.36 from the official repos and follow the instruction from the Readme.md:

KeePass 2.36: install packages sudo apt-get install libmono-system-xml-linq4.0-cil libmono-system-data-datasetextensions4.0-cil libmono-system-runtime-serialization4.0-cil mono-mcs

This works until Http-Connector or KeePass2 will crash, than you will get the same error message until logoff or reboot. Maybe also the KeePassXC (GitHub) could solve the problem. Didn't tried it.

[](https://github.com/pfn/keepasshttp/issues/306)

**Comment by diffusae** [comment]
[Issues 306](https://github.com/pfn/keepasshttp/issues/306)
[keepassxc](https://keepassxc.org)

**Comment by diffusae** [comment]
There is a port of the Http-Connector Plugin: KeePassHttpX optimized for Mac OS X.
Maybe that works?

[KeePassHttp](https://github.com/sazonov/keepasshttpx)

**Comment by diffusae** [comment]
If you only need the KeePassHTTP support, I would recommend you to use [KeePassXC](https://keepassxc.org/project). It seems to be a bit less vulnerable way, to use a password manager.
